custom_components/kia_uvo/KiaUvoApiCA.py

```diff
@@ -188,7 +200,7 @@ def start_climate(self, token: Token):
 
         response = requests.post(url, headers=headers, data=json.dumps({"setting": {"airCtrl": 0, "defrost": "false", "heating1": 0, "igniOnDuration": 3, "ims": 0}, "pin": self.pin}))
         response = response.json()
-
+        time.sleep(3)
```

Comment:
Why do you need this?

Reply:
The API takes a good while to actually function.  This seems to help so we don't move on and call the service again.   I may be able to remove these with the adjustments I just made though.